src/probnum/diffeq/odesolver.py

```diff
@@ -39,13 +39,11 @@ def solve(self, firststep, steprule, **kwargs):
 
             if steprule.is_accepted(stepsize, errorest):
                 self.num_steps += 1
-                t = t_new
-                current_rv = proposed_rv
```

Comment:
Am I super confused or is there no more `current_rv` now? What does the next predict step start with? Do the tests pass? The build seems to fail.

Reply:
Woops, good catch! Big mistake there, I'll fix that